app/views/qsets/show.html.erb

```diff
@@ -24,50 +24,58 @@
         <div class="panel-heading">
           <h4 class="panel-title">
             <div class="row valign">
-              <div class="col-sm-1">
-                <div class="net-votes"><p id="question-<%= question.id %>-net-votes"><%= question.plusminus %></p><br></div>
-              </div>
-              <div class="col-sm-1">
-                <div class="question-voting">
-                  <a class="upvote-button" data-vote="upvote" data-vote-qid="<%= question.id %>">Up</a><br>
-                  <a class="downvote-button" data-vote="downvote" data-vote-qid="<%= question.id %>">Down</a>
+              <div class="col-xs-9 col-sm-9">
+                <div class="col-xs-4 col-sm-3">
+                  <div class="net-votes"><p id="question-<%= question.id %>-net-votes"><%= question.plusminus %></p><br></div>
+                  <div class="question-voting">
+                    <!-- <a class="upvote-button" data-vote="upvote" data-vote-qid="<%= question.id %>">Up</a><br> -->
+                    <div class="glyphicon glyphicon-triangle-top upvote-button" data-vote="upvote" data-vote-qid="<%= question.id %>"></div>
+                    <div class="glyphicon glyphicon-triangle-bottom downvote-button" data-vote="downvote" data-vote-qid="<%= question.id %>"></div>
+                    <!-- <a class="downvote-button" data-vote="downvote" data-vote-qid="<%= question.id %>">Down</a> -->
+                  </div>
                 </div>
-              </div>
-              <div class="col-sm-6">
-                <div style="cursor:default" class="question-content">
-                  <%= link_to question_path(question) do%>
-                  <%= question.text %>
-                  <% end %>
-                  <br/>
-                  <div class="pull-right">
-                      <!-- Creates the edit and delete options for each question -->
-                      <% if can? :manage, question %>
-
-                        <%= link_to edit_question_path(question), class:'btn shortcut-button-link' do%>
-                          <span class="glyphicon glyphicon-edit shortcut-button-glyphicon"></span>
+                <div class="col-xs-8 col-sm-9">
+                  <div style="cursor:default" class="question-content">
```

Comment:
Best practice is to move all style information into CSS and use classes to customize DOM elements; perhaps this `cursor:default` can go in `.question-content`?